helper/identity/identity.go

```diff
@@ -50,7 +50,13 @@ func (e *Entity) Clone() (*Entity, error) {
 
 func (e *Entity) UpsertAlias(alias *Alias) {
 	for i, item := range e.Aliases {
-		if item.ID == alias.ID {
+		// Name with MountAccessor form to be the factors that represent an alias in a
```

Comment:
@ncabatoff alternative IDs is added to both group and entity alias tables due to this function, UpsertAlias, not distinguishing the type of alias we are upserting. Neither do the memdb query functions. If you have any suggestions to how we can approach this, I'm open to hearing.